packages/shorebird_cli/pubspec.yaml

```diff
@@ -11,6 +11,7 @@ environment:
 dependencies:
   archive: ^3.3.6
   args: ^2.3.1
+  barbecue: ^0.4.0
```

Comment:
Reminds me that we will eventually care about providence of our dependencies.  Not yet, but as an SDK that will produce binaries that run on millions (billions!) of devices we will need to care about our supply chain so we're not an eventual EventStream/Solarwinds.

Reply:
Yup it's Apache 2.0 but we can always roll our own or use something else.